src/Indicator.vala

```diff
@@ -330,6 +337,37 @@ public class Session.Indicator : Wingpanel.Indicator {
         current_dialog.set_transient_for (indicator_icon.get_toplevel () as Gtk.Window);
         current_dialog.show_all ();
     }
+
+    private async void update_tooltip () {
+        debug ("Updating tooltip");
```

Comment:
Not sure this kind of debug message is useful in final code although helpful during development.

Reply:
Ahh yeah I'll remove it.